### ISSUE TITLE
Shaman - Searing totem range and aggro bug.

### ISSUE THREAD

**Issue by kysqt** [dalaranwow/dalaran-wow#5262]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: When a shaman places searing totem it in a duel or in general pvp it does not start attacking the nearest target within 20 yards, unless the target is auto attacking or in auto attack stance or range of auto attacks to the shaman, sometimes the totem does not attack at all if placed more than 5 yards from a target attacking the shaman.

**Expected behaviour**: Summons a Searing Totem with 5 health at your feet for 1 min that repeatedly attacks an enemy within 20 yards for 103 to 136 fire damage. <<<Tooltip

**Steps to reproduce the problem**:

1. Make a shaman
2. place searing totem in a duel or pvp, more than 5 yards from enemy or place anywhere near enemy that's not attacking the shaman.
3. 

**Include proofs for this behaviour**
https://gyazo.com/a87e9244200ae959993ff87c76cb5421
https://gyazo.com/afd38e696fc6b721ecd3e24e0895447c

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Issue moved to https://github.com/dalaranwow/dalaran-wow/issues/1311
